src/components/base/link/Link.js

```diff
@@ -91,7 +93,7 @@ function Link({
 
   // Return the component
   return (
-    <NextLink href={href} shallow={true} scroll={scroll}>
+    <NextLink href={href} shallow={true} scroll={scroll} {...props}>
```

Comment:
Sikker på det? Der kan vel være alt muligt crap i den, som ikke skal med... :thinking:

Reply:
Forstår ikke helt den her. Returner vi nogensinde dette NextLink? fordi den får da ingenting med ellers. Ikke engang className. Det burde være et NextLink for alle links der ikke fører væk fra sitet

Reply:
lader værd her, men kan vi gøre det på normal rendering?

Reply:
Ville forvente at alle props der ikke skal ud i DOM, skal fanges?

Reply:
Jeg ved det ikke :shrug: Hvad tror du er bedst?

Jeg er tilbøjelig til at sige at vi på et-eller-andet tidspunkt skal få fjernet props som input fra Link (namedExport-Link, ikke default), for jeg tror vi har fanget dem alle fra de andre props :thinking:

Reply:
I den ideelle verden havde jeg altid bygget sådan nogle base komponenter med en spread - da jeg derfra kan bruge de n til at smide ting i DOM. eks, data-cy eller data-* - vi kan ikke liste dem alle op, så vi giver dem bare videre du ved. 
Tør ikke sige hvad det ville betyde netop her, i et projektet der har kørt så længe. 
Typescript ku hjælpe optimalt, da man kan fx give dem en AnchorHTMLElement type på, og så styre typescript alle dom attributes som supporteres af et <a> tag. Det er ønskescenarier